systemvm/debian/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -471,6 +469,40 @@ def _remove_ipv6_to_interface(self, interface, ipv6):
             logging.info("IPv6 address %s not present for %s" % (ipv6, dev))
             return
 
+    def _enable_radvd(self, dev, ipv6, dns1, dns2):
```

Comment:
@shwstppr 
I think it is unnecessary to remove and regenerate `/etc/radvd.conf`.

for isolated networks, it is generated by /opt/cloud/bin/setup/common.sh
for vpc, it is generated by /opt/cloud/bin/cs/CsVpcGuestNetwork.py

we need to manage (start/stop) service `radvd` in CsRedundant.py

Reply:
@shwstppr 
just my suggestion above.

I have tested a network with RVR, both VRs are BACKUP, no primary ...
however, `/opt/cloud/bin/configure_router.py --primary` in VR looks good.

Reply:
> @shwstppr 
> I think it is unnecessary to remove and regenerate `/etc/radvd.conf`.
> 
> for isolated networks, it is generated by /opt/cloud/bin/setup/common.sh
> for vpc, it is generated by /opt/cloud/bin/cs/CsVpcGuestNetwork.py
> 
> we need to manage (start/stop) service `radvd` in CsRedundant.py

@weizhouapache added this because setting up radvd from common.sh won't work in RVR case. As we discussed earlier, RVRs won't get an IPv6 in guest range from Java code. Therefore, eth0ip6 variable will be empty in common.sh and it won't setup radvd.

So, either we assign an IPv6 to RVRs in guest range to setup radvd with common.sh or change common.sh code to setup radvd based on guest gateway (but in that it won't know which VR is primary) or setup radvd for primary RVR in CsRedundant

Reply:
> @weizhouapache added this because setting up radvd from common.sh won't work in RVR case. As we discussed earlier, RVRs won't get an IPv6 in guest range from Java code. Therefore, eth0ip6 variable will be empty in common.sh and it won't setup radvd.

@shwstppr 
you can get the values in /etc/cloudstack/cmdline.json which has the following
```
    "guestcidr6size": "64", 
...
    "guestgw6": "2022:2:2:e::1", 
```